### DISCUSSION TITLE
how to generate vtk file for radial geometry ?

### DISCUSSION THREAD

**Discussion by medhay36** [acroucher/PyTOUGH#42] [Q&A] [answered]
hello, 

i am new user of pytough.

i tried to follow the tutorials.

generatig vtk file works as shown in rectangular mesh. (3d hydrostatic example)

but for radial geometry example no command was provided to generate the vtk.

can you help me please by adding the command to generate the vtk file in the radial geometry example ?

**Comment by acroucher** [comment, marked as answer]
If you want VTK output from a radial model, you will need to create a dummy rectangular geometry representing a vertical slice through your radial grid. You can then use the same methods for VTK output as in the 3-D case.

For example if `dr` is a list or array of your radial block sizes and `dz` is a list or array of the vertical layer thicknesses, then:

```python
  geo = mulgrid().rectangular(dr, [1.], dz)
```

will create a rectangular geometry with the radial block sizes in the `x` direction and the layer thicknesses in the `z` direction (the `y` direction is a dummy and is not used).

**Reply by medhay36** [reply]
Thank you professor for your answer. It worked perfectly. 

I have another question about post-processing 
I tried to use the t2listing package as shown in the 'horiz1D.listing' example.

But i don't know which output file to use.

In fact, my version of TOUGH generates 2 outputs : a ".out" file and a ".csv" file.

Can you indciate me how to use them with the PyTOUGH package ? 
I actually manipulate them manually.

**Reply by acroucher** [reply]
I am guessing if you get a *.csv file then you're probably running TOUGH3. The other *.out file should be a listing file which you can read using `t2listing`.
